android/src/main/java/com/swmansion/rnscreens/ScreenFragment.kt

```diff
@@ -192,11 +192,12 @@ open class ScreenFragment : Fragment {
         if (fragment is ScreenStackFragment && fragment.canDispatchEvent(event)) {
             fragment.screen.let {
                 fragment.setLastEventDispatched(event)
+                val surfaceId = UIManagerHelper.getSurfaceId(it)
```

Comment:
I believe on Paper `surfaceId` gets resolved always to -1 -- it is Paper compatible.

Reply:
`UIManagerHelper.getSurfaceId` exists [since 0.65](https://github.com/facebook/react-native/commit/62f0dee2353b14ce1524dc62de5e1d2f1883a089) -> that leaves 0.64 behind -> this PR might wait for a while before it is merged. But this situation nudges me towards thoughts that maybe we should reopen the discussion on what should be the minimal supported RN version, because it is 8 versions back right now.